emacs-git update

oscarfv opened the pull request.

Commit:
emacs: update PKGBUILD

Commit:
emacs: removed obsolete patches

Biswa96 commented on mingw-w64-emacs-git/PKGBUILD:
```diff
@@ -1,38 +1,40 @@
 # Maintainer: Haroogan <Haroogan@gmail.com>
 # Maintainer: Oscar Fuentes <ofv@wanadoo.es>
 
+_enable_jit=$([[ "${MINGW_PREFIX}" =~ /mingw.* ]] && echo yes)
```
This check should be done with `MSYSTEM` or `MINGW_PACKAGE_PREFIX` variable.

oscarfv commented:
Ping!

Commit:
emacs: quita -D_FORTIFY_SOURCE, usa -fno-optimize-sibling-calls

Sin esto el binario no funciona correctamente. Específicamente, sin
-D_FORTIFY_SOURCE se producen fallos en las llamadas a procesos.
